openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -51,7 +51,21 @@ def process(self, instance):
         )
         preset = lib.load_capture_preset(data=self.capture_preset)
 
+        capture_presets = openpype.api.get_current_project_settings()["maya"]["publish"]["ExtractPlayblast"]["capture_preset"] # noqa
```

Comment:
Rather use settings from context
```suggestion
        project_settings = instance.context.data["project_settings"]
        capture_presets = project_settings["maya"]["publish"]["ExtractPlayblast"]["capture_preset"] # noqa
```

Reply:
Oh crap, you don't have to get manually settings at all you can just use `self.capture_preset`.

Reply:
I had no idea, thank you!